test/e2e/apimachinery/apigroup_list.go

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apimachinery
+
+import (
+	"context"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+
+	"github.com/onsi/ginkgo"
+)
+
+var _ = SIGDescribe("apigroup list", func() {
+	f := framework.NewDefaultFramework("apigroup-list")
+	ginkgo.It("should locate PreferredVersion for each APIGroup", func() {
+
+		// TEST BEGINS HERE
+		ginkgo.By("[status] begin")
```

Comment:
I don't think it is worth to put this begining message here because the above test name will be output just before this.

Reply:
Agree. Will remove it.

Reply:
Removed